### DISCUSSION TITLE
Extreme slowdown after update from 0.4.8 to 0.5.2

### DISCUSSION THREAD

**Discussion by ColbyBurkett** [open-webui/open-webui#8219] [General]
This morning, I updated by deleting the container & image, then creating new container and letting it pull "main." It pulled 0.5.2 and the container started just fine. UI accessible just fine. Banner about 0.5.2 update w/messaging popped up just fine.

All appeared to be in order. I issued a prompt against llama3.2-3b-fp16, which would normally crank out 90 response tokens/s was abysmally slow at just around 1tkn/s. I investigated all manner of things, including rebooting my Ollama container, upgrading Ollama from 0.5.0 to 0.5.4, thinking I must have some kind of mismatch.... 

Nope

running 'ollama run model' gave the expected performance, so I knew it wasn't model processing/GPU or anything like that. So, I destroyed the container and image, and re-created w/v0.4.8 tag.

All is back to normal.

Environment:

Ollama 0.5.4 on Debian 12 container within Proxmox
Open-webui 0.5.2 (the 'broken' one) under Docker on Synology NAS w/8 cores and 32GB of RAM
Client: Windows 11 24H2 w/Edge browser (my typical browser for accessing things in my home - I have my reasons, lol). Um... yah, didn't try Chrome, Brave, or Canary.... wondering now.

Anyone see anything similar? ideas? Any key piece of info I may have left out?

**Comment by tjbck** [comment]
https://github.com/open-webui/open-webui/discussions/8088#discussioncomment-11683246

**Reply by ColbyBurkett** [reply]
Thank you! I've added a reply there.
